### PR TITLE
updated pricing.md

### PR DESCRIPTION
**Pull request by ZachMassa**
Edited the layout of Quantinuum pricing to be more legible and included the PAYG plan.

### CODE REVIEW

**Comment by Jak-MS** [comment]
@SoniaLopezBravo 
Can you review the proposed changes? 

**Important**: When the changes are ready for publication, adding a `#sign-off` comment is the best way to signal that the PR is ready for the review team to merge. 

#label:"aq-pr-triaged"
@MicrosoftDocs/public-repo-pr-review-team

File: articles/pricing.md [hunk 1]
```diff
@@ -161,45 +161,47 @@ Azure Quantum Credits consumption is based on a resource-usage model and cost of
 > There are no costs or charges for using your free credits. However, there may be some small storage costs, as the input and output of your credits jobs are stored in a storage account that you pay for. Job data is typically <1MB per job.
 > For more details, see [Azure Blob Storage pricing](https://azure.microsoft.com/pricing/details/storage/blobs/).
 
-### [Standard Subscription](#tab/tabid-standard)
+### [H1 Subscriptions](#tab/tabid-H1)
 
-The Standard Subscription is a monthly subscription plan available through queued access.
+The H1 Subscriptions are a monthly subscription plan available through queued access providing access to the H1-1 quantum computer.
```
Comment (SoniaLopezBravo) [inline articles/pricing.md:166]:
```suggestion
The H1 Subscriptions are monthly subscription plans available through queued access providing access to the H1-1 quantum computer.
```

File: articles/pricing.md [hunk 2]
```diff
@@ -161,45 +161,47 @@ Azure Quantum Credits consumption is based on a resource-usage model and cost of
...
+|---|---|
+|USD175,000/Month + Azure infrastructure costs | <ul><li>17k HQCs for use on the System Model H1 hardware</li><li>170k eHQCs for use on the System Model H1 Emulator</li></ul>|
+
 For more information about Azure infrastructure costs, see [Azure Blob Storage pricing](https://azure.microsoft.com/pricing/details/storage/blobs/).
 
-### [Premium Subscription](#tab/tabid-premium)
+### [H1 + H2 Subscriptions](#tab/tabid-H2)
 
-The Premium Subscription is a monthly subscription plan available through queued access.
+The Standard H1 + H2 Subscriptions are a monthly subscription plan available through queued access.
```
Comment (SoniaLopezBravo) [inline articles/pricing.md:180]:
```suggestion
The Standard H1 + H2 Subscriptions are monthly subscription plans available through queued access.
```

File: articles/pricing.md [hunk 3]
```diff
@@ -161,45 +161,47 @@ Azure Quantum Credits consumption is based on a resource-usage model and cost of
...
+### [Pay as You Go](#tab/tabid-PAYG)
 
-The Premium H1 + H2 Subscription is a monthly subscription plan available through queued access.
+The Pay as You Go plan provides queued access to either the H1 series quantum computers or H2 series quantum computers and is charged per HQC usage.
 
-| Pricing | Includes access to  |
+|H1 Pricing| Includes access to |
 |---|---|
-|USD185,000/Month + Azure infrastructure costs| <ul><li>17k HQCs for use on System Model H1 and H2 hardware</li><li>170k eHQCs for use on the System Model H1 or H2 emulators</li></ul>|
+|Per credit cost + Azure infrastructure costs | <ul><li>H1 series quantum computers and emulator|
```
Comment (SoniaLopezBravo) [inline articles/pricing.md:198]:
```suggestion
|Per HQC usage + Azure infrastructure costs | <ul><li>H1 series quantum computers and emulator|
```

**Review by SoniaLopezBravo — APPROVED** [review]
Thanks!

**Pull request merged**